maintenance/wikia/WikiaMaps/removeImapTags.php

```diff
@@ -0,0 +1,258 @@
+<?php
+
+/**
+ * Maintenance script to remove <imap /> tags from articles
+ *
+ * @usage SERVER_ID=203236 php removeImapTags.php --tiles-set-id=1 --verbose --test
+ */
+
+ini_set( "include_path", dirname( __FILE__ )."/../../" );
+
+ini_set('display_errors', 'stderr');
+ini_set('error_reporting', E_ALL);
```

Comment:
I would put `ini_set` statements after the `require_once` to prevent MediaWiki from overriding our settings.